src/base/QXmppJingleIq.cpp

```diff
@@ -288,6 +293,50 @@ void QXmppJingleIq::Content::setTransportPassword(const QString &password)
     d->transportPassword = password;
 }
 
+///
+/// Returns the types and parameters for RTP Feedback Negotiation.
+///
+/// \return the RTP Feedback Negotiation types and parameters
+///
+/// \since QXmpp 1.5
+///
+QVector<QXmppJingleRtpFeedbackTypeAndParameters> QXmppJingleIq::Content::rtpFeedbackTypesAndParameters() const
+{
+    return d->rtpFeedbackTypesAndParameters;
+}
+
+///
+/// Sets the types and parameters for RTP Feedback Negotiation.
+///
+/// \param type RTP Feedback Negotiation types and parameters
+///
+/// \since QXmpp 1.5
+///
+void QXmppJingleIq::Content::setRtpFeedbackTypesAndParameters(const QVector<QXmppJingleRtpFeedbackTypeAndParameters> &rtpFeedbackTypesAndParameters)
+{
+    d->rtpFeedbackTypesAndParameters = rtpFeedbackTypesAndParameters;
+}
+
+///
+/// Returns the intervals for RTP Feedback Negotiation.
+///
+/// \return the RTP Feedback Negotiation intervals
+///
+QVector<QXmppJingleRtpFeedbackInterval> QXmppJingleIq::Content::rtpFeedbackIntervals() const
+{
+    return d->rtpFeedbackIntervals;
+}
+
+///
+/// Sets the intervals for RTP Feedback Negotiation.
+///
+/// \param type RTP Feedback Negotiation intervals
+///
+void QXmppJingleIq::Content::setRtpFeedbackIntervals(const QVector<QXmppJingleRtpFeedbackInterval> &rtpFeedbackIntervals)
+{
+    d->rtpFeedbackIntervals = rtpFeedbackIntervals;
+}
```

Comment:
can you test this by creating a new element using the setters and then serializing it?